backstage/templates/postgresql-ca-config.yaml

```diff
@@ -12,10 +17,9 @@ metadata:
     "helm.sh/hook-delete-policy": "before-hook-creation"
 data:
   {{ .Values.global.postgresql.caFilename }}: |
+{{- if .Values.postgresql.enabled}}
 {{ include "backstage.postgresql.generateCA" . | indent 4}}
 {{- else }}
-{{- $caConfig := printf "%s-postgres-ca"  (include "backstage.fullname" .) }}
-{{- if not ( lookup "v1" "ConfigMap" .Release.Namespace $caConfig ) }}
-{{- fail (printf "\n\nPlease create the '%s' configmap with the CA certificate for your existing postgresql: kubectl create configmap %s --from-file=ca.crt" $caConfig $caConfig) }}
+{{ tpl (.Files.Get "files/rds-ca.crt.tpl") . | indent 4 }}
```

Comment:
I think we should have an explicit conditional to trigger this e.g. else if rds.enabled
Will make it easier to another external db and also cover the case where both are enabled (by mistake probably)